runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -112,18 +160,22 @@ public final class HttpClientFactory implements HttpStreamFactory
                     .build();
 
     private static final String8FW HEADER_AUTHORITY = new String8FW(":authority");
+    private static final String8FW USER_AGENT = new String8FW("user-agent");
     private static final String8FW HEADER_CONNECTION = new String8FW("connection");
     private static final String8FW HEADER_CONTENT_LENGTH = new String8FW("content-length");
     private static final String8FW HEADER_METHOD = new String8FW(":method");
     private static final String8FW HEADER_PATH = new String8FW(":path");
     private static final String8FW HEADER_STATUS = new String8FW(":status");
     private static final String8FW HEADER_TRANSFER_ENCODING = new String8FW("transfer-encoding");
     private static final String8FW HEADER_UPGRADE = new String8FW("upgrade");
+    private static final String8FW HEADER_HTTP2_SETTINGS = new String8FW("HTTP2-Settings");
```

Comment:
Please sort `HEADER_HTTP2_SETTINGS` alphabetically before `HEADER_METHOD`, for consistency.